README.md

```diff
@@ -2,9 +2,9 @@
 Hardware dimensions usable in mechanical design.
 
 ##Function
-Dimensions are stored in Open Document Spreadsheets (ods). CSV and JSON files generated by the build.sh script are available. LibreOffice is used to generate the CSV files.
+Dimensions are stored in  JSON files generated BowlerStudio. 
```

Comment:
I agree JSON is much better for the final application. However I like the ease at which you can do consistency checks with CSV, for example if a field is missing. Speaking of which, this may be good to add to a test suite. A test suite and just JSON may be sufficient to ensure everything is correct.

I prefer to edit the data in a spreadsheet. Are you providing forms to fill in the data?